src/logging/CGlogLoggerAdaptor.cpp

```diff
@@ -48,11 +48,9 @@ void CGlogLoggerAdaptor::Write(bool force_flush,
     
     if (_mylogger && message_len > 0) {
         if (_mutex) {
-            _mutex->lock();
+            _mutex->lock(); // TODO (bgruber): use std::lock_guard
```

Comment:
std::lock_guard is unfortunately not available in C++98. We have to stick with C++98 as long as the DLR uses old 8 years old systems...